src/main/views/magistrates-standard-list.njk

```diff
@@ -0,0 +1,216 @@
+          <li>{{ restrictionBulletPoint1 }}</li>
+          <li>{{ restrictionBulletPoint2 }}</li>
+        </ul>
+        <p class="govuk-body">{{ restrictionInformationP5 }} <a href="#" class="govuk-link">{{ restrictionLink }}</a> </p>
+      </div>
+      {% if bill %}
+        <div class="govuk-body">
+          <h3>{{ restrictionInformationHeadingBill }}</h3>
+          <p class="govuk-body">{{ restrictionInformationP1Bill }}</p>
+          <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+              <span class="govuk-warning-text__assistive">Warning</span>
+              {{ restrictionInformationBoldTextBill }}
+            </strong>
+          </div>
+          <p class="govuk-body">{{ restrictionInformationP2Bill }}</p>
+          <p class="govuk-body">{{ restrictionInformationP3Bill }}</p>
+          <p class="govuk-body">{{ restrictionInformationP4Bill }}</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>{{ restrictionBulletPoint1Bill }}</li>
+            <li>{{ restrictionBulletPoint2Bill }}</li>
+          </ul>
+          <p class="govuk-body">{{ restrictionInformationP5Bill }} <a href="#" class="govuk-link">{{ restrictionLinkBill }}</a> </p>
+        </div>
+      {% endif %}
+    </div>
+  </div>
+  {% for courtList in listData['courtLists'] %}
+    {% set courtHouse = courtList['courtHouse'] %}
+    {% set sessionCount = 0 %}
+    {% for courtRoom in courtHouse['courtRoom'] %}
+      {% for session in courtRoom['session'] %}
+        {% set sessionCount = sessionCount + 1 %}
+        <div class="site-header" tabindex="0" aria-label="Court Details">
+          {% if session['formattedJudiciaries'] | length %}
+            <h1 class="govuk-heading-l site-address" tabindex="0">{{ courtRoom['courtRoomName'] }}: {{ session['formattedJudiciaries'] }}</h1>
+          {% else %}
+            <h1 class="govuk-heading-l site-address" tabindex="0">{{ courtRoom['courtRoomName'] }}</h1>
+          {% endif %}
+        </div>
+        <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+        {% for sitting in session['sittings'] %}
+          {% for hearing in sitting['hearing'] %}
+            {% if hearing['defendantHeading'] | length %}
+              {% for case in hearing['case'] %}
+                  <div class="govuk-accordion__section govuk-accordion__section--expanded">
+                    <div class="govuk-accordion__section-header">
+                      <h2 class="govuk-accordion__section-heading">
+                      <span class="govuk-accordion__section-button" id="accordion-default-heading-{{ sessionCount }}">
+                        {% if hearing['gender'] | length %}
+                          {{ defendantNameHeading }} {{ hearing['defendantHeading'] }} ({{ hearing['gender'] }})
```

Comment:
I think we are missing an 'In Custody' flag here after the gender

Reply:
ok, so that's what asterisk means :) thanks. Added